showcase/src/app/news/news-item-component/news-item.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Inject, Input, OnInit } from "@angular/core";
+import { STARK_LOGGING_SERVICE, /*StarkErrorImpl, */ StarkLoggingService } from "@nationalbankbelgium/stark-core";
+
+@Component({
+	selector: "news-item", // <news-item></news-item>
+	styleUrls: ["./_news-item.component.scss"],
+	templateUrl: "./news-item.component.html"
+})
+export class NewsItemComponent implements OnInit {
+	@Input() release: string;
+
+	@Input() newsDate: string;
+
+	public constructor(@Inject(STARK_LOGGING_SERVICE) public loggingService: StarkLoggingService) {}
+
+	public ngOnInit(): void {
+		this.loggingService.debug("hello from `News` component");
+
+		if (typeof this.release === "undefined" || this.release === "") {
+			console.log(this.release);
```

Comment:
Please remove all console.logs that are not really necessary. If they are, then please use the StarkLoggingService instead ;)